Sources/SwiftKafka/Configuration/KafkaProducerConfig.swift

```diff
@@ -59,6 +59,12 @@ public struct KafkaProducerConfig: Hashable, Equatable {
         set { self.dictionary["message.send.max.retries"] = String(newValue) }
     }
 
+    /// Allow automatic topic creation on the broker when subscribing to or assigning non-existent topics. The broker must also be configured with auto.create.topics.enable=true for this configuration to take effect. Note: the default value (true) for the producer is different from the default value (false) for the consumer. Further, the consumer default value is different from the Java consumer (true), and this property is not supported by the Java producer. Requires broker version >= 0.11.0.0, for older broker versions only the broker configuration applies.
```

Comment:
Can we break this into multiple lines?

Reply:
Broke the documentation comment down into multiple lines ✅

Reply:
I do not see the issue with default values:

In `KafkaProducerConfig` it should be `true`,
in `KafkaConsumerConfig` it should be `false` by default